.github/workflows/changelog-label-check.py

```diff
@@ -0,0 +1,15 @@
+from github import Github
+import os
+import re
+
+
+repo = Github(os.environ.get('GITHUB_TOKEN')).get_repo(os.environ.get('GITHUB_REPOSITORY'))
+pr_number = int(re.search('refs/pull/([0-9]+)/merge', os.environ.get('GITHUB_REF')).group(1))
+pr_labels = repo.get_pull(pr_number).get_labels()
```

Comment:
Does `$GITHUB_EVENT_PATH` also contain the labels ? If so, maybe we don't need `PyGithub `